packages/gatsby/src/bootstrap/load-plugins/load.js

```diff
@@ -159,5 +158,17 @@ module.exports = async (config = {}) => {
     },
   })
 
+  // Add the auto page creator plugin
+  plugins.push({
+    resolve: slash(`${process.cwd()}/node_modules/gatsby-plugin-page-creator`),
+    id: `Plugin gatsby-plugin-page-creator`,
+    name: `gatsby-plugin-page-creator`,
+    // version: `1.0.0`,
```

Comment:
To get the version, just do `require(path.join(require.resolve(`gatsby-plugin-page-creator`), `package.json`)).version